src/key_command/constants.rs

```diff
@@ -77,6 +78,7 @@ cmd_constants![
     (CMD_ZOXIDE, "z"),
     (CMD_ZOXIDE_INTERACTIVE, "zi"),
     (CMD_FLAT, "flat"),
+    (CMD_ESCAPE, "escape"),
```

Comment:
Calling it escape seems a bit ambiguous, is there any reason for this? ie. did ranger call it escape?

I'm thinking it might be better to call it `mode_exit` or `mode_escape`?

wdyt?

Reply:
Hi!
No, ranger does not call it “escape”. Ranger uses this mapping: `map <esc> change_mode normal`. It seems (at least in my old `rc.conf`), that `change_mode` is only used here. For entering visual, Ranger uses `map V  toggle_visual_mode`.

Anyway, my idea for this `escape` command was that it can be used to escape from anywhere. I did not want to be so greedy and use Escape just for exiting visual mode. :smile:  Joshuto cannot handle key-mappings per mode (and I don't think it's necessary), so other features which also activate some kind of “temporary state” could also use the escape-command to “go back to normal”. If we name it something like `mode_escape`, it would be counter-intuitive to use the same command for resetting other things.

Anyway, if you prefer to name it (and use it) only for exiting visual mode, I'm totally ok with that.

Reply:
Okay, in that case, we can keep `escape`.

Maybe in the future, if different modes and different keymappings per mode is added, we can change it.